<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Study Korean</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(230, 230, 230, 75%);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
}

QPushButton:hover:!pressed {
	border-style: outset;
	background-color: rgb(210, 210, 210, 75%);
}

QPushButton:pressed {
	border-style: inset;
	background-color: rgb(180, 180, 180, 75%)
}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="main_window">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>218</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monotype Corsiva</family>
       <pointsize>28</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Study Korean</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>421</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>421</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="study">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>421</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Study</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="study_window">
    <widget class="QLabel" name="question">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>381</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>테스트</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="C">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="A">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="D">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>190</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="add_window">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Enter the Korean phrase along with its English translation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Korean</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>110</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>English</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_word">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_back">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="korean_text">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>145</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="english_text">
     <property name="geometry">
      <rect>
       <x>273</x>
       <y>145</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
